Jellyfin.Networking/Manager/NetworkManager.cs

```diff
@@ -0,0 +1,1273 @@
+                }
+
+                return new NetCollection(_bindAddresses);
+            }
+        }
+
+        /// <inheritdoc/>
+        public bool IsInLocalNetwork(IPObject address)
+        {
+            if (address == null)
+            {
+                throw new ArgumentNullException(nameof(address));
+            }
+
+            if (address.Equals(IPAddress.None))
+            {
+                return false;
+            }
+
+            // See conversation at https://github.com/jellyfin/jellyfin/pull/3515.
+            if (TrustAllIP6Interfaces && address.AddressFamily == AddressFamily.InterNetworkV6)
+            {
+                return true;
+            }
+
+            lock (_intLock)
+            {
+                // As private addresses can be redefined by Configuration.LocalNetworkAddresses
+                return _lanSubnets.Contains(address) && !_excludedSubnets.Contains(address);
+            }
+        }
+
+        /// <inheritdoc/>
+        public bool IsInLocalNetwork(string address)
+        {
+            if (IPHost.TryParse(address, out IPHost ep))
+            {
+                lock (_intLock)
+                {
+                    return _lanSubnets.Contains(ep) && !_excludedSubnets.Contains(ep);
+                }
+            }
+
+            return false;
+        }
+
+        /// <inheritdoc/>
+        public bool IsInLocalNetwork(IPAddress address)
+        {
+            if (address == null)
+            {
+                throw new ArgumentNullException(nameof(address));
+            }
+
+            // See conversation at https://github.com/jellyfin/jellyfin/pull/3515.
+            if (TrustAllIP6Interfaces && address.AddressFamily == AddressFamily.InterNetworkV6)
+            {
+                return true;
+            }
+
+            lock (_intLock)
+            {
+                // As private addresses can be redefined by Configuration.LocalNetworkAddresses
+                return _lanSubnets.Contains(address) && !_excludedSubnets.Contains(address);
+            }
+        }
+
+        /// <inheritdoc/>
+        public bool IsPrivateAddressRange(IPObject address)
+        {
+            if (address == null)
+            {
+                throw new ArgumentNullException(nameof(address));
+            }
+
+            // See conversation at https://github.com/jellyfin/jellyfin/pull/3515.
+            if (TrustAllIP6Interfaces && address.AddressFamily == AddressFamily.InterNetworkV6)
+            {
+                return true;
+            }
+            else
+            {
+                return address.IsPrivateAddressRange();
+            }
+        }
+
+        /// <inheritdoc/>
+        public bool IsExcludedInterface(IPAddress address)
+        {
+            lock (_intLock)
+            {
+                if (_bindExclusions.Count > 0)
+                {
+                    return _bindExclusions.Contains(address);
+                }
+
+                return false;
```

Comment:
```suggestion
                return _bindExclusions.Contains(address);
```

First line of `Contains` function checks for length.